app/src/main/java/com/humberto/tasky/core/presentation/designsystem/components/TaskyPasswordTextField.kt

```diff
@@ -50,10 +51,9 @@ fun TaskyPasswordTextField(
     onTogglePasswordVisibility: () -> Unit,
     hint: String,
     modifier: Modifier = Modifier,
+    isFocused: MutableState<Boolean>,
```

Comment:
Rather than passing in actual `MutableState` fields, I'd just pass their value, like you do with everything else. This is mainly for consistency, but also because the calling composable does not immediately know if and when this state could change (to be fair, with `isFocused` it's kinda obvious, but this might not always be the case). Instead, the typical way to handle this is that you provide a `isFocused: Boolean` as well as `onFocusChange: (FocusState) -> Unit` and then update your state in the composable above or in your VM